Emulator/BF.File.Emulator/Utilities/BfChecker.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FileEmulationFramework.Lib.Utilities;
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BF.File.Emulator.Utilities
```

Comment:
Consider using [File Scoped Namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) in all now code 😉

Reply:
I converted all the code to use them in 2526cc2 and will definitely use them from now on :)